extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js

```diff
@@ -5,13 +5,15 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersRegistry', [
 	'ext.wikia.adEngine.lookup.prebid.adapters.audienceNetwork',
 	'ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',
 	'ext.wikia.adEngine.lookup.prebid.adapters.openx',
+	'ext.wikia.adEngine.lookup.prebid.adapters.rubicon',
 	'ext.wikia.adEngine.lookup.prebid.adapters.wikia',
 	'ext.wikia.adEngine.lookup.prebid.adapters.veles',
 	'wikia.window'
-], function(aol, appnexus, audienceNetwork, indexExchange, openx, wikia, veles, win) {
+], function(aol, appnexus, audienceNetwork, indexExchange, openx, rubicon, wikia, veles, win) {
 	'use strict';
 
 	var adapters = [
+			rubicon,
```

Comment:
Because of 💰 :> (Rubicon in general). But I can move it lower if there is any reason